dashboard/views/includes/header.ejs

```diff
@@ -59,6 +59,9 @@
     <!--  chart.min.js -->
     <script src="/public/js/Chart.min.js"></script>
 
+    <!-- datetimepicker -->
+     <link type="text/css" rel="stylesheet" href="/public/css/jquery.datetimepicker.min.css" />
```

Comment:
5 space for indent is used instead of 4.

Reply:
Don't add the same library in both header and footer. Remove this library import from here.